### PR TITLE
Removed one question, added 3 new.

### PR DESCRIPTION
**Pull request by a-ast**
My suggestion is to remove one of the questions, because the only correct answer to it is 'There are only two classes named ResponseListener in the whole of Symfony and they implement EventSubscriberInterface' is, in the fact, not about Symfony security. 
That's true that one of the Response listeners corresponds to "remember me" functionality, however, it is in lesser degree about security than about forms.
IMHO, the question is too abstract with totally incoherent answers.
Instead of this one I proposed three other questions.

### CODE REVIEW

File: data/security.yml [hunk 1]
```diff
@@ -22,11 +22,22 @@ questions:
             - {value: 'Symfony\Bundle\SecurityBundle\Security\FirewallContext services are synthetically created that correspond to each context you create under your security.yml, each context is liked to an authenticator', correct: false}
             - {value: 'Symfony\Bundle\SecurityBundle\Security\FirewallContext services are publicly created that correspond to each context you create under your security.yml, a context is related to each FirewallContext directly but is overall handled by a generic Symfony\Bundle\SecurityBundle\Security\Firewall object', correct: true}
     -
-        question: 'In Symfony Security component:'
+        question: 'Is user authenticated in all of security firewalls after a successful login:'
+        answers:
+            - {value: 'Yes, it happens autmatically', correct: false}
```
Comment (mickaelandrieu) [inline data/security.yml]:
```diff
- autmatically
+ automatically
```

File: data/security.yml [hunk 2]
```diff
@@ -22,11 +22,22 @@ questions:
...
+            - {value: 'Yes, if firewalls have the same value of the `context` option', correct: true}
+            - {value: 'Yes, if option `shared` is set to true', correct: false}
+            - {value: 'No, it is not possible, firewalls are independent from each other', correct: false}
+    -
+        question: 'How to force a secure area to use the HTTPS protocol in the security config?'
+        answers:
+            - {value: 'access_control: { path: ^/secure, requires_channel: https }', correct: true}
+            - {value: 'access_control: { path: ^/secure, use_https: true }', correct: false}
+            - {value: 'access_control: { path: ^/secure, always_use_https: true }', correct: false}
+            - {value: 'access_control: { path: ^/secure, schemes: [https] }', correct: false}
```
Comment (mickaelandrieu) [inline data/security.yml]: ahahah dat "trap" :p

File: data/security.yml [hunk 3]
```diff
@@ -22,11 +22,22 @@ questions:
...
+    -
+        question: 'What is a purpose of security encoders in security.yml?'
```
Comment (mickaelandrieu) [inline data/security.yml]:
```diff
- a purpose
+ the purpose
```

**Comment by a-ast** [comment]
Fixed.

**Comment by mickaelandrieu** [comment]
Thank you !

**Pull request merged**